infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -506,31 +633,25 @@ private Metadata parseMetadata(FileChannel fileChannel, String segmentFile) thro
         try {
             metadata = Metadata.parseFrom(buf.array());
         } catch (InvalidProtocolBufferException e) {
-            String errorMessage = getDataCorruptionErrorMessage("Can't parse metadata",
-                    fileChannel, segmentFile
-            );
+            String errorMessage = getDataCorruptionErrorMessage(
+                    "Can't parse metadata", segmentFile, fileChannel);
             throw new DataCorruptionException(errorMessage, e);
         }
 
         if (metadata.getLengthChecksum() != Checksum.getChecksum(metadata.getLength())) {
-            String errorMessage = getDataCorruptionErrorMessage("Metadata: invalid length checksum",
-                    fileChannel, segmentFile
-            );
+            String errorMessage = getDataCorruptionErrorMessage(
+                    "Metadata: invalid length checksum", segmentFile, fileChannel);
             throw new DataCorruptionException(errorMessage);
         }
 
         return metadata;
     }
 
-    private String getDataCorruptionErrorMessage(
-            String message, FileChannel fileChannel, String segmentFile) throws IOException {
-        return message +
-                ". Segment File: " + segmentFile +
-                ". File size: " + fileChannel.size() +
-                ". File position: " + fileChannel.position() +
-                ". Global tail: " + logMetadata.getGlobalTail() +
-                ". Tail segment: " + dataStore.getTailSegment() +
-                ". Stream tails size: " + logMetadata.getStreamTails().size();
+    static String getDataCorruptionErrorMessage(String message,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651450)